os/net/mac/tsch/tsch.c

```diff
@@ -862,6 +862,12 @@ PROCESS_THREAD(tsch_send_eb_process, ev, data)
     unsigned long delay;
 
     if(tsch_is_associated && tsch_current_eb_period > 0) {
+#ifdef TSCH_RPL_CHECK_DODAG_JOINED
```

Comment:
why not just add this check to the if and remove the goto statement?

Reply:
I agree your suggestion. I will fix it.